en/installconfig-rack.md

```diff
@@ -37,6 +37,9 @@ immediately in the web UI and begin to sync with the primary controller:
 
 ![add controller][img__add-rackd]
 
+See [MAAS CLI - advanced tasks][cli-list-rackd] for how to list (confirm) all
```

Comment:
*confirm* is a little too abbreviated for me. Maybe just (and confirm) would do it.